features/helpers.rb

```diff
@@ -0,0 +1,23 @@
+# In order to avoid repeating operations supposed to take place x times in various tests, I created this module
+# This article helped a lot: http://blog.pivotal.io/labs/labs/cucumber-step-definitions-are-not-methods
+
+module Helpers
+  def signup(user)
+    visit "/users/sign_out"
```

Comment:
You probably do not need this first visit (in both helpers) or with that in place you're making sure that user is logged out ?